README.md

```diff
@@ -24,6 +24,9 @@ This project is currently in development! It will be announced when there is a (
   GITHUB_ID=<your-github-client-id>
   GITHUB_SECRET=<your-github-client-secret>
   MAILGUN_KEY=<your-mailgun-api-key>
+  EXAMPLES_USERNAME=<your-name>
```

Comment:
these variables need to be added to the production installation .env instructions too!